### PR TITLE
Comment manage and bind external pool ops

### PR DESCRIPTION
**Pull request by artemskriabin**

### CODE REVIEW

File: Stellar-operation-bind-external-system-id.x [hunk 1]
```diff
@@ -46,9 +58,11 @@ struct BindExternalSystemAccountIdSuccess {
     ext;
 };
 
+//: Result of operation applying
```
Comment (khramoff) [inline Stellar-operation-bind-external-system-id.x:61]: khm-khm

File: Stellar-operation-bind-external-system-id.x [hunk 2]
```diff
@@ -46,9 +58,11 @@ struct BindExternalSystemAccountIdSuccess {
...
 union BindExternalSystemAccountIdResult switch (BindExternalSystemAccountIdResultCode code)
 {
 case SUCCESS:
+    //: `success` if used to pass useful fields after operation applying
```
Comment (khramoff) [inline Stellar-operation-bind-external-system-id.x]: is

File: Stellar-operation-manage-external-system-id-pool-entry.x [hunk 3]
```diff
@@ -85,9 +111,11 @@ struct ManageExternalSystemAccountIdPoolEntrySuccess {
     ext;
 };
 
+//: Result of operation applying
```
Comment (khramoff) [inline Stellar-operation-manage-external-system-id-pool-entry.x:114]: khm-khm

File: Stellar-operation-bind-external-system-id.x (hunk 1 above)
Comment (artemskriabin) [reply Stellar-operation-bind-external-system-id.x:61]: )

File: Stellar-operation-manage-external-system-id-pool-entry.x (hunk 3 above)
Comment (artemskriabin) [reply Stellar-operation-manage-external-system-id-pool-entry.x:114]: )

File: Stellar-operation-manage-external-system-id-pool-entry.x [hunk 4]
```diff
@@ -11,18 +11,25 @@ Result: ManageExternalSystemAccountIdPoolEntryResult
 
 */
 
+//: Actions which can be performed with external system account ID pool
```
Comment (khramoff) [inline Stellar-operation-manage-external-system-id-pool-entry.x:14]: performed on

File: Stellar-operation-manage-external-system-id-pool-entry.x [hunk 5]
```diff
@@ -11,18 +11,25 @@ Result: ManageExternalSystemAccountIdPoolEntryResult
...
     REMOVE = 1
 };
 
+//: CreateExternalSystemAccountIdPoolEntryActionInput is used to
+//: pass necessary params to create new external system account ID pool
 struct CreateExternalSystemAccountIdPoolEntryActionInput
 {
+    //: Type of external system, selected arbitrarily
     int32 externalSystemType;
+    //: Data for external system binding
```
Comment (khramoff) [inline Stellar-operation-manage-external-system-id-pool-entry.x:27]: @artemskriabin, isn't it JSON?

File: Stellar-operation-manage-external-system-id-pool-entry.x (hunk 4 above)
Comment (artemskriabin) [reply Stellar-operation-manage-external-system-id-pool-entry.x:14]: no

File: Stellar-operation-manage-external-system-id-pool-entry.x (hunk 5 above)
Comment (artemskriabin) [reply Stellar-operation-manage-external-system-id-pool-entry.x:27]: it can be any string

**Pull request merged**